packages/amplify-category-interactions/commands/interactions/add.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+
+const subcommand = 'add';
+const category = 'interactions';
+const servicesMetadata = JSON.parse(fs.readFileSync(`${__dirname}/../../provider-utils/supported-services.json`));
```

Comment:
Why not use `require` instead of `JSON.parse`

Reply:
check.